src/main/java/org/myrobotlab/service/config/SpeechSynthesisConfig.java

```diff
@@ -4,10 +4,11 @@
 
 public class SpeechSynthesisConfig extends ServiceConfig {
```

Comment:
this class should probably be abstract to make sure no one instantiates it .. it should always have a subclass instance for the specific service that is using it.

Reply:
If I was lazy .... (or smarter) ... I wouldn't have had all the subclassed config  - at the moment all the subclasses do nothing..
because it all handled in the SpeeechSynthesisConfig ... but I was scared and did the stupid thing of subclassing it all, without derived differences